src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -187,7 +187,7 @@
                                 <table mat-table [dataSource]="zaak.gerelateerdeZaken">
                                     <ng-container *ngFor="let column of gerelateerdeZaakColumns" [matColumnDef]="column">
                                         <th mat-header-cell *matHeaderCellDef> {{'gerelateerde.zaak.' + column | translate}} </th>
-                                        <td mat-cell *matCellDef="let row"> {{row[column] | empty}} </td>
+                                        <td mat-cell *matCellDef="let row"> {{column === 'startdatum' ? (row[column] | datum) : (row[column] | empty)}}</td>
```

Comment:
De check kan weg. Je kunt achter empty ook datum erachter pipen. Die datum pipe werkt enkel alleen voor datum formaten, dus zal de andere velden niet in de weg zitten.

dus: {{row[column]} | empty | datum}}

Die empty is volgens mij ook onnodig, want de 5 velden die we tonen voor gerelateerde zaken zijn verplicht :D  Maar laat nog maar even staan.

Reply:
Ik had dit expres losgertokken omdat ik vermoedde dat in een van die velden wel eens een datum zou kunnen zitten, en die parser hem dan ook oppakt. Maar als een gemeente een zaaktype aanmaakt met in de naam een datum gaat er meer mis dan alleen het tonen van wat data van gerelateerde zaken

Ik heb het opgelost in een opvolgende commit - empty pipe overigens laten staan.